Tests/DatadogTests/Datadog/RUM/RUMTelemetryTests.swift

```diff
@@ -266,6 +266,22 @@ class RUMTelemetryTests: XCTestCase {
         XCTAssertEqual(events.count, 0)
     }
 
+    func testSampledTelemetry_rejectAllConfiguration() throws {
+        // Given
+        let telemetry: RUMTelemetry = .mockWith(core: core, sampler: .mockKeepAll(), configurationExtraSampler: .mockRejectAll())
+
+        // When
+        // sends 10 telemetry events
+        for index in 0..<10 {
```

Comment:
CI is complaining that this index isn't used, you can do:
```suggestion
        for _ in 0..<10 {
```